ariadne/cardano/src/Ariadne/Wallet/UiAdapter.hs

```diff
@@ -34,14 +34,15 @@ import Ariadne.Wallet.Cardano.Kernel.DB.Util.IxSet
 import Ariadne.Wallet.Face
 
 import Pos.Core (unsafeIntegerToCoin)
+import Pos.Crypto.Hashing (AbstractHash(..))
 import Serokell.Util (enumerate)
 
 import Control.Lens (makeLenses)
 
 -- 'WalletData' like data type, used only in UI glue
 data UiWalletData = UiWalletData
   { _uwdName     :: !Text
-  , _uwdId       :: !HdRootId
+  , _uwdId       :: !Text
```

Comment:
This field was not used anywhere. Alternative is formatting hash in `Glue`, but that does not depend on cardano-sl-crypto, so I can't import `Pos.Crypto.Hashing` to pattern match `AbstractHash`. I don't think we want cardano dependency in Glue.

Reply:
`Glue` is part of `ariadne-*-app`, which depends on `ariadne-cardano` anyway.
IMHO, backend should spew out as typed values as possible, while frontend should take as generic values as possible.
Glue connects them according to the concrete executable needs.